### DISCUSSION TITLE
Python Django debugging configuration

### DISCUSSION THREAD

**Discussion by g0t0wasd** [mfussenegger/nvim-dap#168] [Q&A]
Hey, there. Thank you for your plugin. Debugging is actually the last thing that keeps me with PyCharm. I wanted to ask if it's possible to set the nvim-dap to work with a Django server? If so, working configuration would be highly appreciated

**Comment by mfussenegger** [comment]
Does https://github.com/mfussenegger/nvim-dap-python/issues/7#issuecomment-755166718 help?
